src/Tizen.NUI.Components/Controls/DropDown.cs

```diff
@@ -513,11 +516,28 @@ public Extents ListMargin
         {
             get
             {
-                return dropDownAttributes.ListMargin;
+                return listMargin;
             }
             set
             {
-                dropDownAttributes.ListMargin = new Extents(value.Start, value.End, value.Top, value.Bottom);
+                dropDownAttributes.ListMargin.CopyFrom(value);
+
+                if (null == listMargin)
+                {
+                    listMargin = new Extents((ushort start, ushort end, ushort top, ushort Bottom) =>
+                    {
+                        dropDownAttributes.ListMargin.Start = start;
+                        dropDownAttributes.ListMargin.End = end;
+                        dropDownAttributes.ListMargin.Top = top;
+                        dropDownAttributes.ListMargin.Bottom = Bottom;
+                        RelayoutRequest();
+                    }, value.Start, value.End, value.Top, value.Bottom);
```

Comment:
Why does only Botton start capital character?

Reply:
It is copy from Extents.cs, I've modified it.